tox.ini

```diff
@@ -0,0 +1,17 @@
+[flake8]
+max-line-length = 99
+doctests = False
+exclude=*build/
+putty-ignore =
+    */__init__.py : +F401
+    docs/conf.py : +E265
+    /^\s*\.\. _.*?: http/ : +E501
```

Comment:
If you can avoid `flake8-putty`, I would. It's not being updated, and imposes compatibility constraints.